docs/source/configuration.rst

```diff
@@ -385,7 +386,7 @@ the tight control of whitespace):
 .. code-block:: cfg
 
     [sqlfluff:templater:jinja:macros]
-    a_macro_def = {% macro my_macro(something) %}{{something}} + {{something * 2}}{% endmacro %}
+    a_macro_def = {% macro my_macro(n) %}{{ n }} + {{ n * 2 }}{% endmacro %}
```

Comment:
Changed `something` to `n` to reduce scrolling (couldn't avoid it completely but mostly readable without scroll now).

Old:
![image](https://user-images.githubusercontent.com/10931297/150039418-c6db4641-5920-4c5b-8dd8-52240ee4f1ce.png)

New:
![image](https://user-images.githubusercontent.com/10931297/150039448-2b3c721c-f7bb-47a4-878f-eac511b1d7d8.png)